units/user-config.target

```diff
@@ -11,3 +11,9 @@ After=user-cloudinit@var-lib-coreos\x2dinstall-user_data.path
 
 Requires=user-cloudinit-proc-cmdline.service
 After=user-cloudinit-proc-cmdline.service
+
+Requires=user-config-vmware-tools.service
```

Comment:
So my motivation here is I can't rely on VMware OEM config to be there at all: in particular it's not there in PXE-booted images on VMware platform. So I was looking for a way to cover all situations where VMware could be the underlying platform, rather than the one where a VMware *image* is used.
Given that, is there another way I could achieve that goal ?